src/js_lexer/identifier.zig

```diff
@@ -108,6 +109,8 @@ pub const JumpTable = struct {
             'A'...'Z', 'a'...'z', '0'...'9', '$', '_' => true,
             else => if (codepoint < 128)
                 return false
+            else if (codepoint == 0x200C or codepoint == 0x200D)
```

Comment:
This also applies for `isIdentifierStart` i think

Reply:
Only identifier continue